fedimint-api/src/db/mod.rs

```diff
@@ -708,6 +708,52 @@ impl DecodingError {
     }
 }
 
+#[macro_export]
+macro_rules! filter_prefixes_by_name {
+    ($prefix:ident, $prefixes:ident) => {
+        if !$prefixes.is_empty() && !$prefixes.contains(&$prefix.to_string().to_lowercase()) {
```

Comment:
This `is_empty` check is completely redundant, no? If so I wonder if there's even a point to have this macro.

Reply:
I removed the entire macro